File: copy_assets.js
const fs = require('fs');
const path = require('path');

const src = 'C:\\Users\\Jabir\\.gemini\\antigravity\\brain\\b6afc95f-db27-4e24-be1f-e7627a614953\\jabx_brand_icon_1774430248669.png';
const dests = [
  'f:\\PieceOfShit\\Projects\\JabX\\public\\icon.png',
  'f:\\PieceOfShit\\Projects\\JabX\\public\\apple-icon.png',
  'f:\\PieceOfShit\\Projects\\JabX\\public\\favicon.ico'
];

dests.forEach(dest => {
  try {
    fs.copyFileSync(src, dest);
    console.log(`Copied to ${dest}`);
  } catch (err) {
    console.error(`Error copying to ${dest}:`, err);
  }
});

File: copy_assets.py
import shutil
import os

src = r"C:\Users\Jabir\.gemini\antigravity\brain\b6afc95f-db27-4e24-be1f-e7627a614953\jabx_brand_icon_1774430248669.png"
public_dir = r"f:\PieceOfShit\Projects\JabX\public"

dests = [
    "icon.png",
    "apple-icon.png",
    "favicon.ico"
]

if not os.path.exists(src):
    print(f"Source file not found: {src}")
    exit(1)

for dest_name in dests:
    dest_path = os.path.join(public_dir, dest_name)
    try:
        shutil.copy2(src, dest_path)
        print(f"Successfully copied to {dest_path}")
    except Exception as e:
        print(f"Error copying to {dest_path}: {e}")
